### ISSUE TITLE
Issue properly reading CSV file

### ISSUE THREAD

**Issue by jplu** [silk-framework/silk#52]
Hello,

Silk has a bug to read standardized CSV files. The bug occur when the columns are between ```"``` to include the possibility to have a ```,``` inside. For example in a CSV file of 14 columns, this row:

```
"279,28868,0,2523,311972526,""MINEOLA COMMUNITY BANK, SSB"",""MINEOLA COMMUNITY BANK, SSB"",215 W BROAD ,MINEOLA,TX,75773,75773,41,2014-10-28T14:04:07"
```

Is counted as a 15 columns row, but it has really 14 columns.

Is it possible to plan a fix?

Thanks in advance.

**Comment by robertisele** [comment]
Could you please post a small CSV file that can be used for reproducing the issue?

**Comment by jplu** [comment]
This seems to be fixed in the 2.7.1 version.

Thanks!
